### PR TITLE
working-groups: add "TiDB Distributions" working group

### PR DESCRIPTION
**Pull request by bb7133**
- Add "TiDB Distributions" working group

### CODE REVIEW

File: working-groups/wg-tidb-distributions.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# TiDB Distributions Working Group
+
+## Goals
+
+This working group aims to build distribution packages of TiDB for different platforms and maintain them for the long-term supports.
+
+## Meetings
+
+* Not needed.
```
Comment (zz-jason) [inline working-groups/wg-tidb-distributions.md]: why?

File: working-groups/wg-tidb-distributions.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Organizers
+
+* [glorv](https://github.com/glorv), PingCAP
+
+## Members
+
+- [bb7133](https://github.com/bb7133), PingCAP
+
+## Contact
+- Slack: [#wg-tidb-distribution](https://tidbcommunity.slack.com/archives/CRGH2NF3J)
```
Comment (gingerkidney) [inline working-groups/wg-tidb-distributions.md]:
```suggestion
- Slack: [#wg-tidb-distributions](https://tidbcommunity.slack.com/archives/CRGH2NF3J)
```

Comment (bb7133) [reply working-groups/wg-tidb-distributions.md]: Addressed, thanks

File: working-groups/wg-tidb-distributions.md (hunk 1 above)
Comment (bb7133) [reply working-groups/wg-tidb-distributions.md]: We've only 2 members close to each other at that time...

Now the team is bigger, we will arrange a meeting time, thanks.

**Comment by gingerkidney** [comment]
please also add the wg into working group readme, the rest lgtm

File: working-groups/wg-tidb-distributions.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# TiDB Distributions Working Group
+
+## Goals
+
+This working group aims to build distribution packages of TiDB for different platforms and maintain them for the long-term supports.
+
+## Meetings
+
+* To be added.
```
Comment (zz-jason) [inline working-groups/wg-tidb-distributions.md]: We can add a simple statement to explain why there is not a meeting, or just remove this section.

File: working-groups/wg-tidb-distributions.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# TiDB Distributions Working Group
```
Comment (zz-jason) [inline working-groups/wg-tidb-distributions.md]: please add this working group to the active working group list in `working-groups/README.md` file.

Comment (bb7133) [reply working-groups/wg-tidb-distributions.md]: Added

File: working-groups/wg-tidb-distributions.md (hunk 3 above)
Comment (bb7133) [reply working-groups/wg-tidb-distributions.md]: I removed this section.

**Comment by bb7133** [comment]
PTAL @zz-jason @gingerkidney, thanks

**Pull request merged**